kahuna/public/js/components/gr-add-label/gr-add-label.html

```diff
@@ -3,19 +3,36 @@
         ng:disabled="ctrl.adding"
         ng:if="!ctrl.active"
         title="Add label">
+
     <gr-icon ng:class="{'spin': ctrl.adding}">add_box</gr-icon>
-        <span ng:hide="ctrl.grSmall">
-            Add<span ng:show="ctrl.adding">ing...</span>
-        </span>
+    <span ng:hide="ctrl.grSmall">
+        Add<span ng:show="ctrl.adding">ing...</span>
+    </span>
 </button>
-<form class="gr-add-label__form" ng:if="ctrl.active" ng:submit="ctrl.addLabel()" >
-    <input class="gr-add-label__form__input" type='text' ng:model="ctrl.newLabel" gr:auto-focus placeholder="label1, label2, label3…" ng:blur="ctrl.cancel()"></input>
+<form
+    class="gr-add-label__form"
```

Comment:
GNU/Emacs:tm: handles both fairly well. I tend to prefer the way we've done it so far for readability reasons, but if it's a pain for all but me to edit those, and the format above works better for all, I don't mind switching to that.